node_vwf.js

```diff
@@ -152,8 +152,8 @@ function startVWF() {
         'destroy buffer size': Infinity, // https://github.com/socketio/socket.io/issues/1592#issuecomment-45885463; should be fixed in socket.io 1.0
     } );
     socketManager.set( 'transports', [ 'websocket' ] );
-    socketManager.set( 'heartbeat timeout', 600 );
-    socketManager.set( 'close timeout', 60 );
+    socketManager.set( 'heartbeat timeout', 3600 ); // ! hour timeout
```

Comment:
Possibly you meant "*1* hour timeout". :-)

Reply:
Are you planning to migrate to socket.io 1.0?  If so, please let me know about it, as I need to make the corresponding update on the HoloLens server side.